### PR TITLE
Update clone perms: allow only template or same committee

### LINKED ISSUE #2564: Allow committee admin to also duplicate meetings from their own committee
**Wanted behaviour:**
If the setting from here https://github.com/OpenSlides/openslides-backend/issues/2507 is activated then the committee's admin should be allowed to duplicate every meeting from this committee and every template into this committee.
Superadmins and Orga-admins should remain capable of duplicating every meeting into every committee

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #2564

### CODE REVIEW

File: docs/actions/meeting.clone.md [hunk 1]
```diff
@@ -39,8 +39,7 @@ If set_as_template is given, template_for_organization_id has to be set to 1.
 
 ## Permission
 
-If a meeting from a different committee is being cloned and said meeting isn't a template, the request user needs the OML `organization.can_manage`
+It is not allowed to clone a meeting from a different committee and said meeting isn't a template.
```
Comment (luisa-beerboom) [inline docs/actions/meeting.clone.md:42]:
```suggestion
It is not allowed to clone a meeting from a different committee if said meeting isn't a template.
```

**Review by luisa-beerboom — APPROVED** [review]
Minor remark, then it is fine by me

File: docs/actions/meeting.clone.md (hunk 1 above)
Comment (reiterl) [reply docs/actions/meeting.clone.md]: done

**Review by Elblinator — APPROVED** [review]
tested

**Pull request merged**